addons/website_slides/security/website_slides_security.xml

```diff
@@ -178,7 +178,7 @@
 
         <!--SLIDE RESOURCE-->
         <record id="rule_slide_slide_resource_downloadable" model="ir.rule">
-            <field name="name">Resource: restricted to channel members and channel responsible</field>
+            <field name="name">Resource: read restricted to channel members</field>
```

Comment:
As we are changing ACLs in stable, would you mind writing some python tests?

There is already a "test_security" file but it does not seem to test the "resource" model.

Also, I would split the "resource" ACL changes from the other small frontend bugfixes if you don't mind.